app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/CategoryTree.php

```diff
@@ -7,35 +7,41 @@
 
 namespace Magento\CatalogGraphQl\Model\Resolver\Products\DataProvider;
 
+use Exception;
 use GraphQL\Language\AST\FieldNode;
-use GraphQL\Language\AST\NodeKind;
 
-use Magento\Store\Api\Data\StoreInterface;
+use GraphQL\Language\AST\NodeKind;
+use Iterator;
 use Magento\Catalog\Api\Data\CategoryInterface;
 use Magento\Catalog\Model\Category;
 use Magento\Catalog\Model\ResourceModel\Category\Collection;
 use Magento\Catalog\Model\ResourceModel\Category\CollectionFactory;
-use Magento\GraphQl\Model\Query\ContextInterface;
 use Magento\CatalogGraphQl\Model\AttributesJoiner;
 use Magento\CatalogGraphQl\Model\Category\DepthCalculator;
+use Magento\CatalogGraphQl\Model\Category\Filter\SearchCriteria;
 use Magento\CatalogGraphQl\Model\Category\LevelCalculator;
 use Magento\CatalogGraphQl\Model\Resolver\Categories\DataProvider\Category\CollectionProcessorInterface;
-use Magento\CatalogGraphQl\Model\Category\Filter\SearchCriteria;
 use Magento\Framework\EntityManager\MetadataPool;
-use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
 use Magento\Framework\Exception\LocalizedException;
-use \Exception;
-use \Iterator;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GraphQl\Model\Query\ContextInterface;
+use Magento\Store\Api\Data\StoreInterface;
 
 /**
  * Category tree data provider
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
 class CategoryTree
 {
     /**
      * In depth we need to calculate only children nodes, so the first wrapped node should be ignored
      */
-    const DEPTH_OFFSET = 1;
+    private const DEPTH_OFFSET = 1;
```

Comment:
I think it should be `public const DEPTH_OFFSET = 1;` because change it to `private`can affect to another extension of third party use this `const` in another class.

Reply:
Hi @edenduong, thank you for your suggest, I updated change in new commit.